### PR TITLE
Add mysql 8.0 CI run

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Comment by rullzer** [comment]
mmm did :boom:... but why...

**Comment by kesselb** [comment]
https://github.com/laradock/laradock/issues/1390#issuecomment-419562297

For some reasons newer MySQL server use pam as primary authentication providder (afaik).

File: .drone.yml [hunk 1]
```diff
@@ -469,7 +469,43 @@ trigger:
...
+  image: nextcloudci/php7.1:php7.1-16
+  commands:
+    - NOCOVERAGE=true TEST_SELECTION=DB ./autotest.sh mysql
+
+services:
+- name: cache
+  image: redis
+- name: mysql
+  image: mysql:8.0
+  command: [ "--default-authentication-plugin=mysql_native_password" ]
```
Comment (kesselb) [inline .drone.yml]: Not sure if this is a proper fix :thinking:

**Comment by rullzer** [comment]
green https://drone.nextcloud.com/nextcloud/server/18145/17/1

File: .drone.yml (hunk 1 above)
Comment (rullzer) [reply .drone.yml]: Does the trick fine by me

**Review by MorrisJobke — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**